src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -3067,6 +3075,16 @@ public boolean claimIsPvPSafeZone(Claim claim)
                 !claim.isAdminClaim() && GriefPrevention.instance.config_pvp_noCombatInPlayerLandClaims;
     }
 
+    public static @Nullable BoundingBox getSpawnProtection(@NotNull World world) {
+        Location spawn = world.getSpawnLocation();
+        int radius = Bukkit.getServer().getSpawnRadius();
+        if (radius == 0) return null;
+        org.bukkit.util.Vector vector = new org.bukkit.util.Vector(16 * radius,0,16 * radius);
```

Comment:
Spawn protection is in blocks, not chunks: https://minecraft.wiki/w/Spawn_protection

Do we need to account for only the default world, or does it apply to all? I honestly have no clue.